app/components/settings/organizations/memberships-count.js

```diff
@@ -0,0 +1,29 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency';
+
+/**
+ * @param organization {UserOrganization} The organization whose memberships we will count
+ */
+export default class SettingsOrganizationsMembershipsCountComponent extends Component {
```

Comment:
I know we don't do this consistently in the app atm, but I'm not a fan of reflecting the route in the component path since it contradicts reuse. Is just `OrganizationMembershipsCountComponent` an option?